test/autorest/generated/lrogroup/lroretrys.go

```diff
@@ -56,6 +73,40 @@ func (client *lroRetrysOperations) BeginDelete202Retry200(ctx context.Context) (
 	}, nil
 }
 
+func (client *lroRetrysOperations) ResumeLroRetrysDelete202Retry200Poller(id string) (LroRetrysDelete202Retry200Poller, error) {
```

Comment:
What would happen if we called this with the token from a different poller type?  We should probably guard against this by embedding the poller type name in the JSON and making sure they match.  It doesn't have to be fixed now if there are more pressing things, just open an issue and we can fix it later.

Reply:
We might also need to include some kind of version number as well to guard against the scenario where there are different SDK versions on each end.

Reply:
Hmmm ok, so I think I'll fix it as soon as I finish the tests. If I manage to finish the tests I'll add that today, if not then Monday morning. Here's the issue #372